packages/frontend/src/utils/project/getContractSection.ts

```diff
@@ -187,7 +212,8 @@ function makeTechnologyContract(
     const tokens = project.config.escrows.find(
       (x) => x.address === item.address,
     )?.tokens
-    if (tokens) {
+    // if contract is an escrow we already tweak it's name so we don't need to add this
```

Comment:
Is this logic really necessary? If I understand correctly it will update the descriptions for contracts that are also inside escrows, but do not follow the newFormat, right?

I feel like this adds additional logic to the already comlpex function and does not provide much value. Maybe let's just assume that we update escrows to the new format as soon as possible, WDYT?